src/WingmanDataTrail/MetricVizPanel/MetricVizPanel.tsx

```diff
@@ -111,6 +112,24 @@ export class MetricVizPanel extends SceneObjectBase<MetricVizPanelState> {
         .build();
     }
 
+    // check if metric is histogram, if yes build heatmap panel
+    const isHistogram = metricName.endsWith('_bucket');
+    if (isHistogram) {
+      return buildHeatmapPanel({
+        panelTitle,
+        headerActions,
+        color,
+        hideLegend,
+        queryRunner: MetricVizPanel.buildQueryRunner({
```

Comment:
I believe the buildQueryRunner function will need to be updated. Is it setting the query option to "heatmap"? Is it passing the function histogram_quantile?

Reply:
ah super helpful - thank you!